MatrixSDK/Data/MXRoomLastMessage.h

```diff
@@ -69,6 +69,8 @@ FOUNDATION_EXPORT NSString *const MXRoomLastMessageDataType;
 
 - (instancetype)initWithEvent:(MXEvent *)event;
 
+- (nullable NSData*)encryptedAttributedString;
```

Comment:
As suggested below, I would rename this `archivedAttributedText` or `attributedTextData` and let it handle both encrypted and unencrypted variant, otherwise we are handling the archiving in two separate objects, depending on the encryption status.

Also I'd convert this into `property (nonatomic, readonly, nullable) NSData *archivedAttributedText` and move just below `attributedText` to make the API more consistent and clearer